x/bank/simulation/operations.go

```diff
@@ -0,0 +1,200 @@
+package simulation
+
+import (
+	"errors"
+	"math/rand"
+
+	"github.com/tendermint/tendermint/crypto"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/simapp/helpers"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/bank/internal/keeper"
+	"github.com/cosmos/cosmos-sdk/x/bank/internal/types"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+)
+
+// SimulateMsgSend tests and runs a single msg send where both
+// accounts already exist.
+func SimulateMsgSend(ak types.AccountKeeper, bk keeper.Keeper) simulation.Operation {
+	return func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accs []simulation.Account) (
+		opMsg simulation.OperationMsg, fOps []simulation.FutureOperation, err error) {
+
+		fromAcc, comment, msg, ok := createMsgSend(r, ctx, accs, ak)
+		if !ok {
+			return simulation.NoOpMsg(types.ModuleName), nil, errors.New(comment)
+		}
+
+		err = sendAndVerifyMsgSend(r, app, ak, msg, ctx, []crypto.PrivKey{fromAcc.PrivKey})
+		if err != nil {
+			return simulation.NoOpMsg(types.ModuleName), nil, err
+		}
+
+		return simulation.NewOperationMsg(msg, true, comment), nil, nil
+	}
+}
+
+func createMsgSend(r *rand.Rand, ctx sdk.Context, accs []simulation.Account, ak types.AccountKeeper) (
+	fromAcc simulation.Account, comment string, msg types.MsgSend, ok bool) {
+
+	fromAcc = simulation.RandomAcc(r, accs)
+	toAcc := simulation.RandomAcc(r, accs)
+	// Disallow sending money to yourself
+	for {
+		if !fromAcc.PubKey.Equals(toAcc.PubKey) {
+			break
+		}
+		toAcc = simulation.RandomAcc(r, accs)
+	}
+	initFromCoins := ak.GetAccount(ctx, fromAcc.Address).SpendableCoins(ctx.BlockHeader().Time)
+
+	if len(initFromCoins) == 0 {
+		return fromAcc, "skipping, no coins at all", msg, false
+	}
+
+	denomIndex := r.Intn(len(initFromCoins))
+	amt, err := simulation.RandPositiveInt(r, initFromCoins[denomIndex].Amount)
+	if err != nil {
+		return fromAcc, "skipping bank send due to account having no coins of denomination " + initFromCoins[denomIndex].Denom, msg, false
+	}
+
+	coins := sdk.Coins{sdk.NewCoin(initFromCoins[denomIndex].Denom, amt)}
+	msg = types.NewMsgSend(fromAcc.Address, toAcc.Address, coins)
+	return fromAcc, "", msg, true
+}
+
+// Sends and verifies the transition of a msg send.
+func sendAndVerifyMsgSend(r *rand.Rand, app *baseapp.BaseApp, ak types.AccountKeeper,
```

Comment:
looking at the diffs for this, it seems this function no longer `verifies the transition of a msg send.`. Maybe change the function name to `sendMsgSend`?